pythoncalculator/subtract.py

```diff
@@ -0,0 +1,2 @@
+def subtract(x, y):
+    return y - y
```

Comment:
Should be subtracting `y` from `x`.
```suggestion
    return x - y
```